debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -611,6 +611,15 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
             .withDescription("The password for the trust store file. "
                     + "Used to check the integrity of the truststore, and unlock the truststore.");
 
+    public static final Field ALLOW_PUBLIC_KEY_RETRIEVAL = Field.create("database.allowPublicKeyRetrieval")
```

Comment:
I don't think wee need the config property for it. It is just `driver.` pass-thorugh property.

Reply:
Thinking about that as it uses `database.` prefix which is not supported for Debezium 2.x is the config option even needed for 8.2 at all?

Reply:
Yes, I have removed the field. The tests are building now. The test failure seems unrelated.